proto/process/agent.proto

```diff
@@ -517,10 +517,10 @@ message Connection {
 	// dns stats based on domain queried, the key corresponds to an index into the `domains` field
 	map<int32, DNSStats> dnsStatsByDomain = 34;
 
-	// http stats summarizing all http transactions recorded, organized by request path
-	map<string, HTTPStats> httpStatsByPath = 35;
-
 	int32 routeIdx = 36;
+
+    // http stats summarizing all http transactions recorded, organized by request path
```

Comment:
Can we mention that this is intended to be a serialized HTTPAggregations object?